libphal/phal_sbe.C

```diff
@@ -326,35 +341,42 @@ void getDump(struct pdbg_target *chip, const uint8_t type, const uint8_t clock,
 	log(level::INFO, "Enter: getDump(%d) on %s", type,
 	    pdbg_target_path(chip));
 
-	if (is_ody_ocmb_chip(chip))
-	{
-		auto ret = sbe_dump(chip, type, clock, faCollect, data, dataLen);
-		if (ret != 0)
-		{
-			log(level::ERROR, "getDump(%s) failed", pdbg_target_path(chip));
+	if (is_ody_ocmb_chip(chip)) {
+		// sbe_ffdc_get internall uses chipop target so probe it
+		pdbg_target *co = get_ody_chipop_target(chip);
```

Comment:
Can we have something like `if (pdbg_target_probe(get_ody_chipop_target(chip)) != PDBG_TARGET_ENABLED)`. Reason for asking is that this is basically a `pdbg_target` object copy in disguise of pointer and the `pdbg_target` can be heavier (having all internal linked lists linked to other lists and all. I mean shallow copy). So if we are not using `co` in anywhere else then we can omit that copying.

Reply:
I retracted some portion of my comments as it was wrong in logic. My sincere apologies.